molgenis-navigator/src/main/frontend/test/unit/specs/components/Navigator.spec.js

```diff
@@ -7,11 +7,11 @@ describe('Navigator', () => {
     })
   })
 
-  describe('submitQuery', () => {
-    it('should clear the query and dispatch a call to fetch all packages', () => {
+  describe('submitQueryWithSpecialCharacters', () => {
+    it('should clear the query and dispatch a call to fetch all packages with special characters', () => {
       Navigator.methods.$store = {
```

Comment:
With the error messages for special characters I would really like to see a test, that checks if the correct error message is given.